cl/recap_email/app/app.py

```diff
@@ -81,11 +81,7 @@ def check_valid_domain(email_address):
 
     tld_domain = domain.split(".")
 
-    # Check if domain (tld_domain[-2]) and tld (tld_domain[-1]) match
-    # with uscourts and gov
-    if tld_domain[-2] != "uscourts" and tld_domain[-1] != "gov":
-        return False
-    return True
+    return tld_domain[-2:] == ["uscourts", "gov"]
```

Comment:
Fixing:

```
cl/recap_email/app/app.py:86:5: SIM103 Return the negated condition directly
   |
84 |       # Check if domain (tld_domain[-2]) and tld (tld_domain[-1]) match
85 |       # with uscourts and gov
86 | /     if tld_domain[-2] != "uscourts" and tld_domain[-1] != "gov":
87 | |         return False
88 | |     return True
   | |_______________^ SIM103
   |
   = help: Inline condition
```

https://docs.astral.sh/ruff/rules/needless-bool/

I think the new version is clear enough that it doesn't need a comment.